apilayer/service/UserService.go

```diff
@@ -0,0 +1,98 @@
+package service
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	stormRider "github.com/earmuff-jam/ciri-stormrider"
+	"github.com/earmuff-jam/ciri-stormrider/types"
+	"github.com/mohit2530/communityCare/db"
+	"github.com/mohit2530/communityCare/model"
+	"github.com/sendgrid/sendgrid-go"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+)
+
+const (
+	EMAIL_SUBJECT_LINE  = "Verify your email address for FleetWise Application"
+	WEB_APPLICATION_URI = "/api/v1/verify/"
+)
+
+// RegisterUser ...
+//
+// Performs saveUser operation and sends email service to the user to verify registration.
+// Creates a random 6 digit token and adds it to the database for the selected user
```

Comment:
comments need to be addressed, our workflow is different from previously expected.